tests/path_gateway_dag_test.go

```diff
@@ -704,34 +703,9 @@ func TestNativeDag(t *testing.T) {
 				Headers(
 					Header("Accept", "text/html"),
 				),
-			Response: Expect().Body(Checks("", func(t []byte) bool {
-				innerCheck := Contains("</html>").Check(string(t))
-				if innerCheck.Success {
-					dagCborHTMLRendering = t
-					return true
-				}
-				return false
-			})),
+			Response: Expect().Body(Contains("</html>")),
```

Comment:
:thought_balloon:  Provides forcing function for implementations to provide meaningful HTMP preview for CBOR in browsers, but without forcing any specific UI or behavior.